Transportation/Bike/BikeHireDockingStation/harvest/bicycle_hire_station_barcelona_harvest.py

```diff
@@ -0,0 +1,169 @@
+#!bin/python
+# -*- coding: utf-8 -*-
+
+"""
+Harmonises data from the city of Barcelona corresponding to
+the bicycle hiring stations
+"""
+
+from __future__ import print_function
+import urllib2
+import json
+from datetime import datetime
+from pytz import timezone
+import re
+
+
+# Origin of the Data (Barcelona's open data)
+source = 'http://wservice.viabicing.cat/v2/stations'
+
+status_dictionary = {
+    'OPN': 'working',
+    'CLS': 'outOfService'
+}
+
+barcelona_tz = timezone('CET')
+
+MIME_JSON = 'application/json'
+FIWARE_SERVICE = 'Bicycle'
+FIWARE_SERVICE_PATH = "/Barcelona"
+
+DATA_BROKER = 'http://localhost:1026'
+
+"""
+See
+# http://fiware-datamodels.readthedocs.io/en/latest/
+Transportation/Bike/BikeHireDockingStation/doc/spec/index.html
+"""
+
+BIKE_STATION_TYPE = 'BikeHireDockingStation'
+
+# Sanitize string to avoid forbidden characters by the Orion Broker
+
+
+def sanitize(str_in):
+    return re.sub(r"[<(>)\"\'=;-]", "", str_in)
+
+
+# Reads the data from the data source and returns a dictionary
+def read_data():
+    req = urllib2.Request(url=source)
```

Comment:
Remove: `urllib2.` to align with the imports.  Repeat this removal on lines 50, 53, 57, 141, 144

Also you can simplify lines 51-61 down to:
```python
import contextlib  # move this to the top of the file

try:
    with contextlib.closing(urllib.urlopen(req)) as f:
        return f.read()  # return json data
except urllib2.URLError as e:
    print('Error while calling: %s : %s' % (source, e))
    return None
```